src/amo/components/SearchSort/index.js

```diff
@@ -0,0 +1,79 @@
+import classNames from 'classnames';
+import React, { PropTypes } from 'react';
+import { compose } from 'redux';
+
+import SearchSortLink from 'amo/components/SearchSortLink';
+import translate from 'core/i18n/translate';
+
+import './SearchSort.scss';
+
+
+export class SearchSortBase extends React.Component {
+  static propTypes = {
+    _searchToggle: PropTypes.object.isRequired,
+    filters: PropTypes.object.isRequired,
+    i18n: PropTypes.object.isRequired,
+    pathname: PropTypes.string.isRequired,
+  }
+
+  constructor(props) {
+    super(props);
+    this.state = { sortVisible: false };
+  }
+
+  onClick = (event) => {
+    event.preventDefault();
+    this.toggleSort(this.props._searchToggle || this.searchToggle);
+  }
+
+  sortOptions() {
+    const { i18n } = this.props;
+
+    return [
+      { sort: 'updated', text: i18n.gettext('Recently Updated') },
+      { sort: 'relevance', text: i18n.gettext('Relevance') },
+      { sort: 'users', text: i18n.gettext('Most Users') },
+      { sort: 'rating', text: i18n.gettext('Top Rated') },
+    ];
+  }
+
+  toggleSort(searchToggle) {
+    // const searchToggle = _searchToggle || this.searchToggle;
+    // Blur the search sort toggle if we're closing it.
+    if (this.state.sortVisible) {
+      searchToggle.blur();
+    }
+    this.setState({ sortVisible: !this.state.sortVisible });
+  }
+
+  render() {
+    const { filters, i18n, pathname } = this.props;
+    const { sortVisible } = this.state;
+    const currentSort = filters.sort || 'relevance';
+
+    return (
+      <div className={classNames('SearchSort', {
+        'SearchSort--visible': sortVisible,
+      })}>
+        <a className="SearchSort-toggle" href="#SearchSortOptions"
+          ref={(ref) => { this.searchToggle = ref; }} onClick={this.onClick}>
+          {i18n.gettext('Sort')}
```

Comment:
Actually, because the icons here already require CSS moving about and moving to Icons everywhere would be a better bug IMO (I'm happy to move to them, but it's more than just here we'd need to), can we just file a bug to move every icon to an `<Icon>` component if that makes sense?